src/common/cockpitwebcertificate.c

```diff
@@ -111,6 +111,41 @@ cockpit_certificate_locate (char **error)
   return NULL;
 }
 
+/**
+ * cockpit_certificate_key_path:
+ *
+ * Check if there is a matching *.key file for the given *.cert file.
+ *
+ * Return key file path on success, or %NULL if there is no matching *.key file.
+ */
+char *
+cockpit_certificate_key_path (const char *certfile)
```

Comment:
No `*at()`-style `dirfd` argument here?

Reply:
We don't need it in the actual callers, neither of them has a dirfd. We could add it and add a test for it in test-webcertificate.c, but we wouldn't use it in practice.

Reply:
This comment is obsolete now, as it's now pure string manipulation.